### DISCUSSION TITLE
https://github.com/LibreChat-AI/librechat.ai ?

### DISCUSSION THREAD

**Discussion by kohsuke** [danny-avila/LibreChat#4496] [Q&A] [answered]
Hi, I recently noticed there's another GitHub org and repository that claims to be librechat.ai: https://github.com/LibreChat-AI/librechat.ai

Can you clarify what's going on? Was there a fork? A dispute about the ownership of the name? Something else?

**Comment by tsutsu3** [comment]
I'm also curious about this.

Has there been any official announcement? 🤔

**Reply by rubentalstra** [reply, marked as answer]
@kohsuke thats the main website 🤓
